content/scalers/apache-kafka-topic.md

```diff
@@ -19,15 +19,19 @@ This specification describes the `kafka` trigger for Apache Kafka Topic.
   triggers:
   - type: kafka
     metadata:
-      brokerList: kafka.svc:9092
+      # brokerList: kafka.svc:9092 - deprecated
+      bootstrapServers: kafka.svc:9092
       consumerGroup: my-group
       topic: test-topic
       lagThreshold: '5'
 ```
 
 **Parameter list:**
 
-- `lagThreshold` Optional. How much the stream is lagging on the current consumer group. Default is 10.
+- `bootstrapServers`: comma separated list of Kafka brokers "hostname:port" to connect to for bootstrap.
```

Comment:
One last request, can you add `brokerList` as well please? As mentioned, we should still document it, but mark it as deprecated.

Reply:
Done ;-)

Reply:
Thanks!